src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -500,3 +508,55 @@ def prepare_logs_argument(c):
             c.argument('app_name', type=str, help="The Azure Spring Cloud app name to configure the route.")
             c.argument('routes_json', type=str, help="The JSON array of API routes.", validator=validate_routes)
             c.argument('routes_file', type=str, help="The file path of JSON array of API routes.", validator=validate_routes)
+
+    for scope in ['spring-cloud build-service builder buildpack-binding create']:
+        with self.argument_context(scope) as c:
+            c.argument('type',
+                       arg_type=get_enum_type(v20220101_preview_AppPlatformEnums.BindingType),
+                       help='Required type for buildpack binding.')
+            c.argument('properties',
+                       help='Non-sensitive properties for launchProperties. Format "key[=value]".',
+                       nargs='*',
+                       validator=validate_buildpack_binding_properties)
+            c.argument('secrets',
+                       help='Sensitive properties for launchProperties. '
+                            'Once put, it will be encrypted and never return to user. '
+                            'Format "key[=value]".',
+                       nargs='*',
+                       validator=validate_buildpack_binding_secrets)
+            c.argument('name', help='Name for buildpack binding.', validator=validate_buildpack_binding_not_exist)
+            c.argument('builder_name', help='The name for builder.', default="default")
+            c.argument('service', service_name_type, validator=only_support_enterprise)
+
+
+    for scope in ['spring-cloud build-service builder buildpack-binding set']:
```

Comment:
It's better to merge `spring-cloud build-service builder buildpack-binding` **create** with **set** except the argument `name`.

Reply:
One question, is this the convention to merge such kinds of statements? If so, I can align this.
As a beginner to CLI, I intended to write parms like this, because it is clean when I look into one command for parameters. I can have a full picture of all parameters of one command.
Compare to merging, I may need to combine these in my mind for full picture.

Could you please help to enlighten me what is the best practice here?

Reply:
Hi Li,
We highly recommend merging, because after merging, the similarities and differences can be more clearly distinguished.
If you modify it later，it is not easy to miss anything after merge.

Reply:
sure, updated.